### PR TITLE
Wrap describe command

### LINKED ISSUE #1494: No Examples by 'osc describe --help'
`osc desccribe --help` should show some command Examples like other operations. It is unclear the number of arguments with current help.

***Current***
~~~
# osc describe --help
Show details of a specific resource.

This command joins many API calls together to form a detailed description of a
given resource.

Use "osc --help" for a list of all commands available in osc.
Use "osc options" for a list of global command-line options (applies to all commands).
~~~

***TOBE***
~~~
# osc describe --help
Show details of a specific resource.

This command joins many API calls together to form a detailed description of a
given resource.

Examples:

	# SOME EXPLANATION
	$ osc describe service docker-registry

	# SOME EXPLANATION
         $ osc describe dc router

Use "osc --help" for a list of all commands available in osc.
Use "osc options" for a list of global command-line options (applies to all commands).
~~~

### PR DESCRIPTION
**Pull request by kargakis**
Closes #1494

### CODE REVIEW

File: pkg/cmd/cli/cmd/wrappers.go [hunk 1]
```diff
@@ -159,3 +159,22 @@ Examples:
...
+This command joins many API calls together to form a detailed description of a
+given resource.
+
+Examples:
+
+	# Provide details about the ruby-20-centos7 image repository
+	$ %[1]s describe imageRepository ruby-20-centos7
+
+	# Provide details about the ruby-sample-build build configuration
+	$ %[1]s describe buildConfig ruby-sample-build
```
Comment (mfojtik) [inline pkg/cmd/cli/cmd/wrappers.go]: I would be one example to use the shortcut (bc instead of buildConfig)

**Comment by mfojtik** [comment]
one nit, LGTM

File: pkg/cmd/cli/cmd/wrappers.go (hunk 1 above)
Comment (kargakis) [inline pkg/cmd/cli/cmd/wrappers.go]: :ok:

**Comment by smarterclayton** [comment]
[merge]

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**